test/unittests/test_compat_routers.py

```
@@ -0,0 +1,183 @@
+    return FakeModel()
+
+
+def _make_app(model) -> FastAPI:
+    from ovos_stt_http_server.routers.openai_whisper import make_openai_whisper_router
+    app = FastAPI()
+    app.include_router(make_openai_whisper_router(model))
+    return app
+
+    app = FastAPI()
+    app.include_router(make_openai_whisper_router(model))
+    app.include_router(make_deepgram_router(model))
```

Ruff (0.15.12)

[error] Undefined name `make_deepgram_router`

(F821)